### ISSUE TITLE
Text "Step of" and Skip are in English

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#290]
Samsung Galaxy J3 Prime
Android version 7.0

Text “Step of" is in English on the French screens. Further, on the Step 6 screen, text "Skip" is in English.

Expected result
The text should be in French.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts

Screenshots
![text](https://user-images.githubusercontent.com/12256067/86048690-ee6b0e00-ba1e-11ea-80c2-2b2268969f0b.gif)
![skip](https://user-images.githubusercontent.com/12256067/86138676-c29e6580-babc-11ea-88b3-c274e5d76d8e.gif)

**Comment by smcmurtry** [comment]
Closed by #318
